rundmcmc/chain.py

```diff
@@ -54,15 +54,20 @@ def __next__(self):
                 else:
                     continue
 
-            self.state.parent = None
+            # Please delete this comment
```

Comment:
:2nd_place_medal:

Reply:
Lol! That was from my "editing on my phone in the GitHub editor" phase, when I could create new lines but not delete any